### PR TITLE
update docker image when new commits pushed to released branch

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: docker
 
 on:
```
Comment (Shylock-Hg) [inline .github/workflows/docker.yml:3]: The release job had do this.

Comment (yixinglu) [reply .github/workflows/docker.yml:3]: yeah. This PR will solve the issue which we created a version branch and push new commits to this branch before releasing.

**Review by yixinglu — APPROVED** [review]
LGTM

File: .github/workflows/docker.yml (hunk 1 above)
Comment (jievince) [reply .github/workflows/docker.yml:3]: It's triggered after the release branch is created and before the released tag is created. During this time, if new commits pushed to the released branch, the docker image should be updated in time.

**Review by dutor — APPROVED** [review]

**Pull request merged**